__tests__/commands/_helpers.js

```diff
@@ -81,9 +81,21 @@ export async function run<T, R>(
 
   let cwd;
   if (fixturesLoc) {
-    const dir = path.join(fixturesLoc, name);
+    let parentDir = name;
+    let childDir = '';
+
+    // if the name is a path, copy the parent dir, then use the child dir as the cwd
+    // there's some existing tests which reference fixtures in a parent dir (../), so excluding those
+    const nameSepIndex = name.indexOf('/');
+    if (nameSepIndex !== -1 && name.indexOf('../') === -1) {
```

Comment:
Agreed it's a little weird. It works with multiple separators (copies the top-most dir, uses deepest dir as the cwd).

To avoid dual-purposing the name field, I could allow `name` to have the type: `string | { source: string, cwd: string }`, e.g. 

```js
return runInstall({}, { source: 'offline-mirror-configuration', cwd: 'enabled-from-parent' }, async (config, reporter) => {
  // ...
});
```

Or, could allow changing the cwd from within the test body, but it'd require another callback as the cwd is required for setup.